docs/docs/getting-started.server.mdx

```diff
@@ -346,14 +346,10 @@ If you use more modern JavaScript features than what your users support,
 [configure Vite’s
 `build.target`](https://vitejs.dev/guide/build.html#browser-compatibility).
 
-<details>
-  <summary>Possible Compatibility Issue</summary>
-  As for [vite](https://github.com/vitejs/vite) 3, `rollup@^2.79.1` is applied.
-  You might notice that there was a warning of peer dependency problem. Please
-  ignore it or add `rollup` manually to your devDependencies with 
-  **version `^2.79.1`**. This avoids possible incompatibility, for when the 
-  version of `rollup` is not specified, `rollup@>=3` will be install by default.
-</details>
+<Note type="info">
+  **note**: Vite 3 requires the older `rollup@<3`. Make sure `rollup@3` is not
+  installed.
```

Comment:
```suggestion
  **note**: Vite 3 requires the older `rollup@2`.
  Make sure `rollup@3` is not installed.
```

a) it specifically needs v2
b) Fix formatting

Otherwise good!

Reply:
Sorry, I've just turned to vscode for linting possible format issues. And sadly a force push to erase typo 😂

Reply:
All is well 😜